common/static/common/js/components/views/search_field.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * A search field that works in concert with a paginated collection. When the user
+ * performs a search, the collection's search string will be updated and then the
+ * collection will be refreshed to show the first page of results.
+ */
+;(function (define) {
+    'use strict';
+
+    define(['backbone', 'underscore', 'text!common/templates/components/search-field.underscore'],
+        function (Backbone, _, searchFieldTemplate) {
+            return Backbone.View.extend({
+
+                events: {
+                    'submit .search-form': 'performSearch',
+                    'blur .search-form': 'performSearch',
+                    'click .action-clear': 'clearSearch'
+                },
+
+                initialize: function(options) {
+                    Backbone.View.prototype.initialize.call(this, options);
+                    this.type = options.type;
+                    this.label = options.label;
+                },
+
+                render: function() {
+                    this.$el.html(_.template(searchFieldTemplate, {
+                        type: this.type,
+                        searchString: this.collection.searchString,
+                        searchLabel: this.label
+                    }));
+                    return this;
+                },
+
+                performSearch: function(event) {
+                    var searchField = this.$('.search-field'),
```

Comment:
Declare dependency on jQuery (for some reason, we have left that out very often in the teams JS files... I'm adding a bunch in with my PR).